TodoApp/TodoApp.Api/Models/Item.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace TodoApp.Api.Models
+{
+    public class Item
+    {
+        [Key]
+        public string Id { get; set; }
+
+        [MinLength(1, ErrorMessage = "Text length must be greater than 1.")]
+        public string Text { get; set; }
+
+        public override bool Equals(object obj)
```

Comment:
Methods Equals and GetHashCode are not necessary. They just override the default behavior of these methods.